corral/main.pony

```diff
@@ -40,11 +40,26 @@ actor Main
           CommandSpec.leaf(
             "init",
             "Initializes the corral.json and dep-lock.json files with"
-              + " skeletal information.")?
+              + " skeletal information.",
+            [
+              OptionSpec.string(
+                "path",
```

Comment:
I wonder if "dir" or "directory" would be a better name for this option. Yes, it's a path, but is it a file path or a directory path? So then, the Help would read like "The directory where corral.json and dep-lock.json will be created."

Reply:
Totally agree. Renamed "path" to "directory".